recipes/simpleconf/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-simpleconf" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-simpleconf-{{ version }}.tar.gz
```

Comment:
Did you have issues when this was `{{ name }}`?

Reply:
no issues

Reply:
Ah, okay, I see this is how Grayskull left it anyway.